lib/person.rb

```diff
@@ -0,0 +1,30 @@
+require 'yaml'
+require 'date'
+require './lib/library.rb'
+
+def books
+  YAML.load_file('./lib/books.yml')
+end
+
+class Person
+  attr_accessor :name, :book
+
+  def initialize(name)
+    @name = name
+    @book = []
+  end
+
+  def set_renter(obj)
+    [:renter] == @name
```

Comment:
This is not right syntax. Please explain what you want to do. The method name indicate that this should be a method in the Library class. But I'm not sure.